status-im/status-react
# Added some new DApps

fixes #6615

### Summary:

Added some new DApps

### Steps to test:
- Open Status
- Verify new DApps are accessible

status: ready
